packageJsonAnalyzer/types.ts

```diff
@@ -0,0 +1,22 @@
+export type CodemodHash = string & { __type: 'CodemodHash' };
+export type PathHash = string & { __type: 'PathHash' };
+
+export type Path = {
```

Comment:
what's the reason for this type?

Reply:
that is the path type that includes the codemodes